canonical-web-and-design/maas-ui
# feat(ui): add network table checkboxes

## Done

- Add checkboxes to the network table in machine details.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Visit the network tab in machine details for a new or ready machine.
- Check that there are checkboxes and that they work as expected.
- Visit a deployed machine and check that the checkboxes are disabled.

## Fixes

Fixes: canonical-web-and-design/maas-squad#2284.
